packages/google_maps_flutter/google_maps_flutter_android/README.md

```diff
@@ -70,8 +70,14 @@ AndroidMapRenderer mapRenderer = AndroidMapRenderer.platformDefault;
   }
 ```
 
-Available values are `AndroidMapRenderer.latest`, `AndroidMapRenderer.legacy`, `AndroidMapRenderer.platformDefault`.
-Note that getting the requested renderer as a response is not guaranteed.
+NOTE: `AndroidMapRenderer.platformDefault` corresponds to `AndroidMapRenderer.latest`.
+
+WARNING: `AndroidMapRenderer.legacy` is known to crash apps and is no longer supported.
```

Comment:
Maybe "no longer supported in the Google Maps SDK"? Or do we mean specifically that we won't investigate issues related to it?

Reply:
Google Maps won't fix any bugs in it and is working towards completely removing it. We should not spend time on it either.